app/controllers/page_mount_controller.rb

```diff
@@ -17,6 +17,8 @@ def proxy
     # ... or determine if having blank cookies means "don't set anything buddy"
     session[:proxied_cookies] = response.cookies unless response.cookies.blank?
 
+    store_client_type_id_in_cms_session(response, request)
```

Comment:
It's technically the same thing (delegation), poor wording in the method definition from myself :(